### DISCUSSION TITLE
Disclosure Triangles for List View

### DISCUSSION THREAD

**Discussion by ketonik** [spacedriveapp/spacedrive#2292] [Feature Request]
Please make List View usable by providing disclosure triangle on each folder so that it reveals, displays the files inside any or all sub folders....obviously the window globally can sort the attributes of files and folders by date, size etc in the same window. Naturally folder sizes should be calculated as a requirement.

ATTENTION WINDOWS USERS, this is NOT Windows 'Tree View', that useless thing cannot display files in more than one sub folder, It displays files in ONLY the selected folder and those files are in a totally different pane, not the same window...again useless.

**Comment by utkubakir** [comment]
This is planned and will be worked on before release.

**Comment by ketonik** [comment]
Any update to share with us on this 'essential' UI functionality? thx

**Comment by ketonik** [comment]
Hi, just wanted to ask if there is a branch you guys can point me to, so to have goosey gander at progress on this?
This is super duper desperately needed for Windows OS. IDK if anyone really understands the daily torture using that OS all day without a single usable file manager.
